src/Level.cpp

```diff
@@ -106,5 +106,5 @@ Level Level::ParseLevel(const std::filesystem::path& path, Level::LandType landT
 
 bool operator<(const Level& first, const Level& second)
 {
-	return first.GetName() < second.GetName();
+	return (first.GetName() < second.GetName());
 }
```

Comment:
Get rid of this. It doesn't make sense other than sorting and we only sort in the UI.